crt_portal/static/sass/_uswds-theme-color.scss

```diff
@@ -24,53 +24,53 @@ Theme palette colors
 */
 
 // Base colors
-$theme-color-base-family:           'gray-cool';
+$theme-color-base-family:           'gray'; // gray-cool
 $theme-color-base-lightest:         'gray-5';
-$theme-color-base-lighter:          'gray-cool-10';
-$theme-color-base-light:            'gray-cool-30';
-$theme-color-base:                  'gray-cool-50';
+$theme-color-base-lighter:          'gray-warm-10'; // gray-cool-10
+$theme-color-base-light:            'gray-30'; // gray-cool-30
+$theme-color-base:                  'gray-50'; // gray-cool-50
 $theme-color-base-dark:             'gray-cool-60';
-$theme-color-base-darker:           'gray-cool-70';
+$theme-color-base-darker:           'gray-70'; // gray-cool-70
```

Comment:
We really don't use the cool grays anywhere (although it seems like some of the USWDS base elements use `$theme-color-base-dark` -- maybe that should be updated to a warm gray also?)

Reply:
:thinking: Can we remove these `gray-...` comments and add an explanation as to why we are still keeping `gray-cool-60` for now?

Reply:
good idea! I'll add that, thanks!